Documenting two examples for `83865`; Adds `order` field to example headers (#436)
haskellfoundation/error-message-index: message-index/messages/GHC-83865/terms/index.md

---
title: Values of Different Types
order: 0
---

Function `inc` has type `Int -> Int`, hence it expects an argument
of type `Int`; yet, on the definition of `two` it was called with
an argument of type `String`.

If you ever need to know the type of something, you can ask for it in `ghci`

> and ask for it with `:type` (or its shorthand `:t`):

Reviewer: ```suggestion
with the command `:type` (or its shorthand `:t`):
```
